.eslintignore

```diff
@@ -0,0 +1,5 @@
+**/node_modules/*
```

Comment:
I'm noticing you didn't create the /ab-<NAME> folder. I'm not sure if in these scenarios we ask students to re-submit. I'll get back to you as soon as possible :)

Reply:
Thank you! I also thought that the "Write all of your code in a directory named lab- + <your name> e.g. lab-susan" instruction meant that I needed to work in a folder titled lab-katherine, but was told that I need to work in a branch titled lab-katherine instead. Please go ahead and check with Duncan, though.